### PR TITLE
feat: Migrate schedules

### LINKED ISSUE #15: Migrate Schedules
- [x] File structure -> based on path name, e.g /openapi/paths/actor-builds
- [x] Fix operationId - make it camelCased - e.g GetListOfKeyValueStores
- [x] File name of endpoints -> remove v2@ e.g. actor-tasks@{actorTaskId}@input.yaml

### PR DESCRIPTION
**Pull request by HonzaTuron**
Closes #15

### CODE REVIEW

File: openapi/components/schemas/schedules/GetListOfSchedulesResponse.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-title: Getlistofschedulesresponse
+title: GetListOfSchedulesResponse
 required:
   - data
 type: object
 properties:
   data:
     allOf:
-      - $ref: ./data24.yaml
+      - $ref: ./ScheduleFilters.yaml
```
Comment (janbuchar) [inline openapi/components/schemas/schedules/GetListOfSchedulesResponse.yaml:8]: the `allOf` could be removed, or are you planning to do that all at once?

Comment (HonzaTuron) [reply openapi/components/schemas/schedules/GetListOfSchedulesResponse.yaml:8]: Yeah probably. Didn't touch this on previous PRs.

File: openapi/components/schemas/schedules/Schedule.yaml [hunk 2]
```diff
@@ -54,5 +54,5 @@ properties:
   actions:
     type: array
     items:
-      $ref: ./actions1.yaml
```
Comment (janbuchar) [inline openapi/components/schemas/schedules/Schedule.yaml:57]: any chance you could rename this file to something better?

File: openapi/components/schemas/schedules/ScheduleShort1.yaml [hunk 3]
```diff
@@ -50,5 +50,5 @@ properties:
   actions:
     type: array
     items:
-      $ref: ./actions.yaml
+      $ref: ../actions.yaml
```
Comment (janbuchar) [inline openapi/components/schemas/schedules/ScheduleShort1.yaml:53]: Also this please :slightly_smiling_face:

File: openapi/components/schemas/schedules/ScheduleCreate.yaml [hunk 4]
```diff
@@ -28,6 +28,6 @@ properties:
   actions:
     type: array
     items:
-      $ref: ./actions2.yaml
+      $ref: ../actions2.yaml
```
Comment (janbuchar) [inline openapi/components/schemas/schedules/ScheduleCreate.yaml:31]: and this

**Review by janbuchar — APPROVED** [review]

**Pull request merged**